programs/ssl/CMakeLists.txt

```diff
@@ -45,7 +45,8 @@ foreach(exe IN LISTS executables)
         ${extra_sources})
     set_base_compile_options(${exe})
     target_link_libraries(${exe} ${libs} ${CMAKE_THREAD_LIBS_INIT})
-    target_include_directories(${exe} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../../framework/tests/include
+    target_include_directories(${exe} PRIVATE ${MBEDTLS_FRAMEWORK_DIR}/tests/programs
```

Comment:
Spotted through the review of #9988, lines 55 and 56:
```
        target_include_directories(${exe}
            PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../test)
```
can be removed it seems, no include file in `programs/test` anymore.